master/src/main/scala/org/apache/celeborn/service/deploy/master/Master.scala

```diff
@@ -79,6 +79,10 @@ private[celeborn] class Master(
   private val authEnabled = conf.authEnabled
   private val secretRegistry = new MasterSecretRegistryImpl()
 
+  if (conf.logMasterConfEnabled) {
+    logInfo(getConf)
```

Comment:
for security purposes, we must redact sensitive data in logging, you can refer to `org.apache.spark.util.Utils#redact`

Reply:
@SteNicholas the redact policy is also applicable if we display configuration in Web UI, otherwise, sensitive data like MySQL password is likely to be leaked to normal users